src/app/shared/components/sidenav-menu/sidenav-menu.component.html

```diff
@@ -13,6 +13,7 @@
       expanded: menu.expanded
     }"
     class="menu-list-item"
+    *ngIf="!menu.authorizedPersona || menu.authorizedPersona.includes(this.appServ.loggedInUserInfo?.persona)"
```

Comment:
nice.

Reply:
It was for this change, I added the `loggedInUserInfo` set to null on logout. Logging in and logging out of admin would result in retaining the admin menus post logout also.